mission/functions/core/ui/taskroster/fn_tr_characterInfo_show.sqf

```diff
@@ -24,26 +24,30 @@ VN_TR_CHARINFO_CTRL ctrlShow true;
 
 ///////////// WIP /////////////
 
-VN_TR_CHARINFO_NAME_CTRL ctrlSetText profileName;
-VN_TR_CHARINFO_POINTS_CTRL ctrlSetText str(player getVariable ["vn_mf_db_rank",0]);
-VN_TR_CHARINFO_RANK_CTRL ctrlSetText (rank player);
+VN_TR_CHARINFO_NAME_CTRL ctrlSetText format ["Profile Name: %1", profileName];
+VN_TR_CHARINFO_POINTS_CTRL ctrlSetText format ["Current Points: %1", str(player getVariable ["vn_mf_db_rank",0])];
+VN_TR_CHARINFO_RANK_CTRL ctrlSetText format ["Current Rank: %1", (rank player)];
 
 _player_id = player getVariable ["vn_mf_db_serial","0"];
-VN_TR_CHARINFO_SNUM_CTRL ctrlSetText _player_id;
+VN_TR_CHARINFO_SNUM_CTRL ctrlSetText format ["Serial Number: %1", _player_id];
+VN_TR_CHARINFO_UID_CTRL ctrlSetText format ["Player UID: %1", getPlayerUID player];
```

Comment:
potentially useful for dac cong RP during interrogations etc. players can give their actual serial number and player UID